packages/contracts/contracts/optimistic-ethereum/ovm/FullStateManager.sol

```diff
@@ -19,7 +19,8 @@ contract FullStateManager is StateManager {
 
     mapping(address=>mapping(bytes32=>bytes32)) ovmContractStorage;
     mapping(address=>uint) ovmContractNonces;
-    mapping(address=>address) ovmCodeContracts;
+    mapping(address=>address) codeContractOf;
+    mapping(address=>address) ovmContractOf;
```

Comment:
Would name these in a way that indicates what the key is. Maybe `ovmAddressToContractAddress` and `contractAddressToOvmAddress`?